packages/sample-app/sample_app/mcp_dev_assistant_server.py

```
@@ -0,0 +1,387 @@
+            "platform": platform.platform(),
+            "architecture": platform.architecture(),
+            "processor": platform.processor(),
+            "python_executable": sys.executable,
+            "current_directory": str(Path.cwd()),
+            "environment_variables": {
+                k: v for k, v in os.environ.items() 
+                if not any(secret in k.lower() for secret in ['key', 'token', 'password', 'secret'])
+            }
+        }
+        
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)